cluster/addons/fluentd-gcp/fluentd-gcp-image/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For systems without journald
+mkdir -p /var/log/journal
+
+if [ -e /host/lib/libsystemd* ]
```

Comment:
fluend container starts, but it is not able to send logs from journald (silently ignores because of missing plugin). So the logs from kubelet.service and docker.service are not sent to cloud logging anymore. 
I did not look at the test in detail, but it seems it only verifies logs from "containers" are exported. It doesn't test whether logs from the system journal are exported.
I will open a new issue for this.